easybuild/easyconfigs/s/STEAK/STEAK-master-foss-2019b.eb

```diff
@@ -0,0 +1,40 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+# You will need to download the tarball manually and put it in the folder
+# where this file is. 
+
+easyblock = 'MakeCp'
+
+name = 'STEAK'
+version = 'master'
+
```

Comment:
This needs a `versionsuffix = "-Python-%(pyver)s"`

Reply:
Thanks. I did not realize this.